### ISSUE TITLE
Tags and sprites conflict in Rel (update: sprites have white background, should be transparent)

### ISSUE THREAD

**Issue by biaji** [plantuml-stdlib/C4-PlantUML#310]
Sample:

```
@startuml
!include <C4/C4_Component>
!include <material/common>
!include <material/close_circle>
AddRelTag("oth", $textColor="white")

System(all, "all"){
Component(test,"test","")
Component(test1,"test1","")
}
Rel(test, test1, "sfef", $tags="oth", $sprite="ma_close_circle")
@enduml
```

![image](https://github.com/plantuml-stdlib/C4-PlantUML/assets/2725285/06da9245-26c9-44f9-8d6c-6690446479f9)

Which results in a white square instead of spirte itself.   If remove the tags param, it displayed as normal

**Comment by kirchsth** [comment]
Hi @biaji,

good finding, I have to check all details (over the weekend) but I think its a bug in PlantUML (transparent part of the Rel() symbol should be displayed transparent and not white)

I could offer only a workaround until I have a final solution:
You can color a sprite (independent of the Rel() color), e.g. with `$sprite="ma_close_circle,color=red"` 

[![](http://www.plantuml.com/plantuml/svg/VO_1IiGm48RlUOeuU6YXi2piqIuhVGFrNcQJsMxe92d951NntKrRaKN4onpco__dcrCMJ39xLdSk69ujwOVkiEiEVHVz50C5ETuWZqB98UzCz3w6ln77JBrnoJ0TbSgZ2nCczFgCPXnId8Fj8iUaXqGKb7gqzedu1OSAeboXqVT2Rx8-QU7rweIM7Rj0tpkBQJppJ50hzVoUXNo5p8s6Cg7-K3_cbL2M1fO93K3zY-mtj2zi-I5FoGcruB6_FQ74FbKHtChq5jIGBtHPNN78xIB-JqLZLl54jfITADZPynS0)](http://www.plantuml.com/plantuml/uml/VO_1IiGm48RlUOeuU6YXi2piqIuhVGFrNcQJsMxe92d951NntKrRaKN4onpco__dcrCMJ39xLdSk69ujwOVkiEiEVHVz50C5ETuWZqB98UzCz3w6ln77JBrnoJ0TbSgZ2nCczFgCPXnId8Fj8iUaXqGKb7gqzedu1OSAeboXqVT2Rx8-QU7rweIM7Rj0tpkBQJppJ50hzVoUXNo5p8s6Cg7-K3_cbL2M1fO93K3zY-mtj2zi-I5FoGcruB6_FQ74FbKHtChq5jIGBtHPNN78xIB-JqLZLl54jfITADZPynS0)

BR Helmut

**Comment by biaji** [comment]
> Hi @biaji,
> 
> good finding, I have to check all details, but I think its a bug in PlantUML (transparent part of the Rel() symbol should be displayed transparent and not white).
> 
> I could offer only a workaround until I have a final solution: You can color a sprite (independent of the Rel() color), e.g. with `$sprite="ma_close_circle,color=red"`
> 


Cool~  That's it!  It should be transparent background as you said.

**Comment by kirchsth** [comment]
@biaji: PlantUML team implemented a fix that it works starting with version [v1.2023.9beta3](https://github.com/plantuml/plantuml/releases/tag/snapshot), details see https://forum.plantuml.net/17926

PS.: the PlantUML picture above is cached for 5 days, therefore I used a minor changed Plantuml text that the updated image with the fix is displayed

[![](https://www.plantuml.com/plantuml/png/RSx1IWCn40RWUvuY3XvsOQ4KUdHB9M-WtfUGJDl0JAOaiwY8xsuqA4Lw-IyVC_z_gEgBhapcBkL0QqJxu7ORjrkSy4KoPjrV4Nl5aZnjWZ1Blaqa5PUGIY3S6_COunFIYpyD87g6oTuhlgaJaZB3wpafmcZCytjLvC4JJHPQmlXX_XOCYbKd-4wO0CP_iksqxVPfMbs_i5qjr2CUVvhzgSw_C-gbjF8Ps2_NazkB0-QuCdq1)](https://www.plantuml.com/plantuml/uml/RSx1IWCn40RWUvuY3XvsOQ4KUdHB9M-WtfUGJDl0JAOaiwY8xsuqA4Lw-IyVC_z_gEgBhapcBkL0QqJxu7ORjrkSy4KoPjrV4Nl5aZnjWZ1Blaqa5PUGIY3S6_COunFIYpyD87g6oTuhlgaJaZB3wpafmcZCytjLvC4JJHPQmlXX_XOCYbKd-4wO0CP_iksqxVPfMbs_i5qjr2CUVvhzgSw_C-gbjF8Ps2_NazkB0-QuCdq1)
